src/components/TextCard/TextCard.tsx

```diff
@@ -108,6 +108,15 @@ export const TextCard = ({
         <RichText className="cc-text-card__description">{description}</RichText>
       )}
       {children && <div className="cc-text-card__description">{children}</div>}
+      {date && type === 'job_card' && (
+        <p className="cc-text-card__closing-date">
+          Closing date:
```

Comment:
Minor: let move the `&nbsp;` before `FormattedDate` to add space.
```js
<p className="cc-text-card__closing-date">
          Closing date:
          <span>
            &nbsp;
            <FormattedDate dateString={date} />
          </span>
</p>
```

Reply:
Thanks - amended!